cypress/e2e/audio_element/spec.cy.ts

```diff
@@ -0,0 +1,14 @@
+describe("audio", () => {
+  before(() => {
+    cy.intercept("/project/settings").as("settings");
+    cy.visit("http://127.0.0.1:8000");
+    cy.wait(["@settings"]);
+  });
+
+  it("should be able to display an audio element", () => {
+    cy.get("#welcome-screen").should("exist");
+
+    cy.get(".message").should("have.length", 1);
+    cy.get(".message").eq(0).find(".inline-audio").should("have.length", 1);
+  });
+});
```

Comment:
[this](https://github.com/cypress-io/cypress/issues/1750) looks interesting to actually test the audio

Reply:
I've found a way to load the audio file, wait for it to be loaded and then check that it was loaded correctly ✅